lib/anycable/rails.rb

```diff
@@ -2,7 +2,6 @@
 
 require "anycable"
 require "anycable/rails/version"
-require "anycable/rails/config"
```

Comment:
What if leave this `require`?

We load `anycable-rails `only in the context of a Rails app, so we can sure that at the time of the `anycable/config` load the Rails app is present.

Reply:
Ok. Done. Now we don't need to mess with initializers